metrics/metrics.go

```diff
@@ -0,0 +1,115 @@
+package metrics
+
+import (
+	"context"
+	"net"
+	"time"
+
+	"github.com/lucas-clemente/quic-go/logging"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+// Measures
+var (
+	connections = stats.Int64("connections", "number of QUIC connections", stats.UnitDimensionless)
+)
+
+// Tags
+var (
+	keyPerspective, _ = tag.NewKey("perspective")
```

Comment:
Not sure about this, but another option here could be encoding server vs client in the counter name:

- /quic/server/connections
- /quic/server/var-that-only-makes-sense-for-servers
- /quic/client/connections
- /quic/client/var-that-only-makes-sense-for-clients

Reply:
Let's keep it as is for now. I'm just beginning to explore what we want to export, so this will probably change a couple of times anyway as we define more metrics.